tox.ini

```diff
@@ -103,7 +103,7 @@ commands =
     echo "COVERAGE: {env:COVERAGE:}"
     cov: coverage run \
     !cov: python \
-        -m pytest -v {env:PYTEST_PATH:} --color=yes --basetemp={envtmpdir} \
+        -m pytest {env:PYTEST_PATH:} --color=yes --basetemp={envtmpdir} \
```

Comment:
Seems unrelated... Why?

Reply:
Because with almost 4k tests, the logs are unreadable. Finding test that fails is harder than in non verbose mode.

Reply:
Can we make the change in a separate PR?

Reply:
done #6996

Reply:
#6996 Thanks!